src/grafana/grafana_launcher.star

```diff
@@ -113,3 +140,30 @@ def new_datasource_config_template_data(prometheus_url):
 
 def new_dashboard_providers_config_template_data(dashboards_dirpath):
     return {"DashboardsDirpath": dashboards_dirpath}
+
+
+def new_additional_dashboards_data(plan, additional_dashboards):
+    data = []
+    for index, dashboard_src in enumerate(additional_dashboards):
+        additional_dashboard_folder_name = "-".join(
```

Comment:
can we take out `-` as constant!

Reply:
If you change `GRAFANA_ADDITIONAL_DASHBOARDS_FOLDER_NAME` to `GRAFANA_ADDITIONAL_DASHBOARDS_FOLDER_NAME = "grafana-additional-dashboards-{0}"` then you can do

`GRAFANA_ADDITIONAL_DASHBOARDS_FOLDER_NAME.format(index)` instead of the join